packages/flutter_test/lib/src/binding.dart

```diff
@@ -1179,6 +1179,8 @@ abstract class TestWidgetsFlutterBinding extends BindingBase
     RendererBinding.instance.initMouseTracker();
     // ignore: invalid_use_of_visible_for_testing_member
     ServicesBinding.instance.resetInternalState();
+    // ignore: invalid_use_of_visible_for_testing_member
+    WidgetsBinding.instance.resetInternalState();
```

Comment:
Doesn't this now execute the reset twice now?

Reply:
Formally no. This code prints different values:

```
class WidgetsBinding {
  static String instance = 'WidgetsBinding';
}


class ServicesBinding extends WidgetsBinding {
  static String instance = 'ServicesBinding';
}

void main() {
  print(WidgetsBinding.instance);
  print(ServicesBinding.instance);  
}
```

Practically maybe in some cases. But there may be other cases (for example in some tests). Will it hurt to do it twice?

Reply:
Removed one and added assert to make suer binding is still as expected by this code.